tensorflow/python/ops/array_ops.py

```diff
@@ -99,6 +99,21 @@
 listdiff = gen_array_ops.list_diff
 
 
+def rank(input, name=None):
+  """Override for gen_array_ops.rank(), making it work for SparseTensor.
```

Comment:
Please copy and properly format the existing documentation [here](https://www.tensorflow.org/versions/r0.8/api_docs/python/array_ops.html#rank).  Additionally, there's no need to mention override.